config/settings.yml

```diff
@@ -1009,6 +1009,7 @@ directory:
 
 va_mobile:
   url: "https://veteran.apps.va.gov"
+  ppms_base_url: "https://staff.apps.va.gov"
```

Comment:
I'm open to renaming this something other than `ppms_base_url` if anyone has better ideas.

Reply:
That's a good question and I'll have to ask @simiadebowale for clarification on that. As you can see, we use a production URL directly above it as well. I believe this only impacts local development and tests. There seem to be a few places in this file where we use production urls. Perhaps they should all be changed to staging and the VCR cassettes updated to match.